### ISSUE TITLE
redis container does not support password

### ISSUE THREAD

**Issue by spicy-sauce** [testcontainers/testcontainers-python#187]
This can be achieved by:
        return DockerContainer(
            image="redis:latest").with_bind_ports(
            6379, redis_port).with_command(f"redis-server --requirepass {redis_password}")

Cannot be achieved in RedisContainer as its @wait_container_is_ready waits to a redis client to be ready only by host and port:
```
  host=self.get_container_host_ip(),
  port=self.get_exposed_port(6379),
```

I can provide a PR, just let me know that it's something you'll be willing to merge.

Thanks.

**Comment by tillahoffmann** [comment]
Yes, a PR would be great, thank you!

**Comment by spicy-sauce** [comment]
https://github.com/testcontainers/testcontainers-python/pull/195

Please review and run checks.
Thanks!

**Comment by yakimka** [comment]
Closed by #195
